package.json

```diff
@@ -45,7 +45,7 @@
     "mocha": true
   },
   "scripts": {
-    "pretest": "retire -n && jshint test/build.test.js test/s3_setup.test.js test/versioning.test.js",
+    "pretest": "retire -np && jshint test/build.test.js test/s3_setup.test.js test/versioning.test.js",
```

Comment:
Yes, because of the security issue with Growl. the `-p` checks only the dependencies